### ISSUE TITLE
Official Pounding Unban Vote.

### ISSUE THREAD

**Issue by ImMikeu** [vincentzhang96/CommunityDiscord#8]

**Comment by ImMikeu** [comment]
Vara votes no

**Comment by Shs21** [comment]
I vote no, as the reason why we banned Jon was in order to prevent the leakage of political chat from occurring in lounge. 

If you have spent time with Jon in the last few months, namely, Otium's discord group. You will know Jon is more expressive & intense in his nature of political discussions than he was before. He also has a secluded channel (which we already attempted to do in the past, and resulted in the removal of the channel & Jon) which is used frequently by Jon, but does not contain him there.

I think this vote, if successful, will result in a regression in what we have accomplished. That is, managing to settle the flames he had burning for several months in the past.

**Comment by ImMikeu** [comment]
I agree with Shs and have altered my vote to no. Even if we brought him back we don't even have a room to quarantine him in. He can't be trusted as far as I'm concerned.

**Comment by grey21man** [comment]
i believe that if we want pounding in, he needs to have the official nsfw politics channel else its gonna be crazy again. they go together

**Comment by Marcmallow** [comment]
I vote no for Pounding unban.

I would like to add the fact that he was spoken to several times (iirc Ael tried handling that those times) and they all didn't last long before he started stuff up again. I don't trust him to follow the rules if we allow him back once again.
